lib/rules/web/admin_console/4.2/deployment.xyaml

```diff
@@ -30,3 +30,31 @@ search_image:
   action: clear_input_value
   action: set_input_value
   action: click_search_button
+deploy_search_nonexisted_image:
+  params:
+    search_content: non-existed
+    content: Could not load image metadata
+  action: search_image
+  action: check_page_contains
+deploy_search_existed_image:
```

Comment:
I think this can be totally replaced with `search_image` it's not necessary to check the message for the positive testing